app/src/main/res/values/strings_help.xml

```diff
@@ -5,43 +5,32 @@
     <string name="help_title">Help</string>
     <string name="help_navigation_title">Navigation</string>
     <string name="help_navigation_text">
-        Swipe down from the home screen to show favorites, and upwards to hide them.
+        Swipe 🡇 from the home screen to show all apps, and 🡇 to go back.
         \n
-        \nSwipe up to show all apps if the favorites panel is hidden.
-        Swipe down from the top of it to return to the home screen.
+        \nSwipe 🡇 from the home screen to show favorites, and 🡅 to go back.
         \n
-        \nThe button below the favorites panel opens the launcher menu.
+        \nClick 🔧 when showing favorites to open the menu.
         \n
         \nLong-hold an app or folder to access its settings.
     </string>
     <string name="help_notification_title">Notification</string>
     <string name="help_notification_text">
         Shows a popup of favorites above other apps when you leave the home screen.
-        \n
-        \If instead it brings you back to the home screen:
-        \n- Press the \"clear all\" button in \"Recent Apps\" or
-        \n- Reboot your device
-        \n
-        \nUnintrusively, the notification icon is transparent and of low priority.

```

Comment:
Why remove the paragraphs about the transparent icon and the case of switching to another launcher?
I think there were useful and now there is only one line in the notification section...